src/app-e-correlation-test-summary-workspace/app-e-correlation-test-summary-workspace.controller.ts

```diff
@@ -143,6 +161,10 @@ export class AppendixETestSummaryWorkspaceController {
     description:
       'Deletes a workspace Appendix E Correlation Test Summary record',
   })
+  @AuditLog({
+    label: 'Deleted appendix E correlation test summary by ID for test summary',
+    requestParamsOutFields: ['locId', 'testSumId']
```

Comment:
According to the ticket, this should return the `id`, too:
```suggestion
    requestParamsOutFields: ['id', 'locId', 'testSumId']
```